chsdi/templates/htmlpopup/cadastralwebmap_opendata.mako

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
```

Comment:
Instead you could inherit the other cadastral webmap mako or make the other one inherit this one.